gaphor/storage/storage.py

```diff
@@ -188,7 +186,7 @@ def _load_elements_and_canvasitems(
     elements, factory, modeling_language, gaphor_version, update_status_queue
 ):
     def create_canvasitems(diagram, canvasitems, parent=None):
-        """Canvas is a read gaphas.Canvas, items is a list of
```

Comment:
Same thing here as well, would now be a good time to rename this function from `canvasitems` to `diagramitems`?

Reply:
Updated.